js/src/carousel.js

```diff
@@ -390,10 +390,12 @@ const Carousel = (($) => {
 
         if (typeof config === 'number') {
           data.to(config)
-
-        } else if (action) {
+        } else if (typeof action === 'string') {
```

Comment:
because ``action`` is always a string.

But if you prefer I can restore the old condition